src/status_im/utils/image_server.cljs

```diff
@@ -0,0 +1,28 @@
+(ns status-im.utils.image-server
+  (:require [quo.design-system.colors :as colors]
+            [status-im.utils.config :as config]))
+
+(def ^:const image-server-uri-prefix "https://localhost:")
+(def ^:const identicons-action "/messages/identicons")
+(def ^:const account-images-action "/accountImages")
+(def ^:const contact-images-action "/contactImages")
+
+(defn current-theme []
+  (case @colors/theme-type
```

Comment:
i would abstract image server from colors , and pass it as a parameter , but its up to you

Reply:
emm.. i thought image-server would be much easier to use without passing param "theme"